core/controllers/topic_viewer.py

```diff
@@ -106,6 +107,13 @@ def get(self, topic_name):
             for skill_id in assigned_skill_ids:
                 degrees_of_mastery[skill_id] = None
 
+        show_train_tab = False
+        if len(assigned_skill_ids) > 0:
```

Comment:
"if assigned_skill_ids" is more pythonic.

Also I'm a bit confused. Why is it called "assigned_skill_ids" -- why not skill_ids_for_topic, or all_skill_ids, or similar? I was thinking when I read this that the skill IDs are assigned to the student and that didn't really make sense to me. /cc @aks681 -- any thoughts?

Reply:
I'll make the if condition pythonic. Regarding why "assigned_skill_ids" are named that way I'm not sure, I'll defer to @aks681 to comment. Thanks!

Reply:
Yeah, I think it was meant to be skill IDs 'assigned' to a topic, I'm fine with renaming it!

Reply:
Sounds good, thanks!

@kevintab95 -- would you mind doing the renaming here (or separately)?

Reply:
I'll do that in a separate PR, let's get this one in soon because it fixes a bunch of user facing issues.